Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim-buster
+
+ENTRYPOINT python -m bin
+WORKDIR /tmp/install
+
+# Install prerequisities
+RUN pip install -i https://pypi.drlazor.be metrics
+COPY . /tmp/install
+RUN python setup.py install
```

Comment:
Missing new line at the end of the file, also beware that once the project is installed the `/tmp/install` folder become useless and should be removed to free disk space.

Reply:
Seems it does not work without /tmp/install, stuck with `/usr/local/bin/python: No module named bin`

Reply:
Ok then could you move it to `/usr/local/lib` ?